researchstack-sdk/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
 
 dependencies {
   compile fileTree(include: ['*.jar'], dir: 'libs')
-  compile 'org.researchstack:skin:1.1.1', {
+  compile project(':skin'), {
```

Comment:
This reminds me that I need to find a build setup that works for both this repo itself, and when it's included as a submodule. Making a JIRA